contracts/pcv/morpho/MorphoCompoundPCVDeposit.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import {ILens} from "./ILens.sol";
+import {IMorpho} from "./IMorpho.sol";
+import {CoreRef} from "../../refs/CoreRef.sol";
+import {PCVDeposit} from "../PCVDeposit.sol";
+import {ICompoundOracle, ICToken} from "./ICompound.sol";
+
+/// @notice PCV Deposit for Morpho-Compound V2.
+/// Implements the PCV Deposit interface to deposit and withdraw funds in Morpho
+/// Liquidity profile of Morpho for this deposit is fully liquid for USDC and DAI
+/// because the incentivized rates are higher than the P2P rate.
+/// Only for depositing USDC and DAI. USDT is not in scope.
```

Comment:
Could add a require() in the constructor with token address = mainnet USDC or mainnet DAI ?

Reply:
This contract should also work for CETH, but USDT has some things that break certain assumptions that bug hunters use to try to get bounties, so I just specifically wanted to call out that this deposit doesn't support USDT.